stories/assets/js/viewport.js

```diff
@@ -9,15 +9,10 @@ $.fn.isInViewport = function () {
   return $elementBottom > $viewportTop && $elementTop < $viewportBottom;
 };
 
-export default function viewport(ele) {
-  $(window).on('resize scroll load', () => {
-    $(ele).each(function () {
-      if ($(this).isInViewport()) {
-        $(this).addClass('inviewport');
-      }
-    });
+$(window).on('resize scroll load', () => {
+  $('.left-right').each(function () {
```

Comment:
Similar to the above, remove class `left-right` and handle this using `data-attribute`.